gensim/parsing/preprocessing.py

```diff
@@ -125,13 +327,63 @@ def stem_text(text):
 
 
 def preprocess_string(s, filters=DEFAULT_FILTERS):
+    """Take string, apply list of chosen filters to it, where filters are methods from this module.
+    Default list of filters consists of: :func:`~gensim.parsing.preprocessing.strip_tags`,
+    :func:`~gensim.parsing.preprocessing.strip_punctuation`, :func:`~gensim.parsing.preprocessing.strip_multiple_whitespaces`,
+    :func:`~gensim.parsing.preprocessing.strip_numeric`, :func:`~gensim.parsing.preprocessing.remove_stopwords`,
+    :func:`~gensim.parsing.preprocessing.strip_short`,  :func:`~gensim.parsing.preprocessing.stem_text`.
+    <function <lambda>> in signature means that we use lambda function for applying methods to filters.
+
+    Parameters
+    ----------
+    s : str
+    filters: list, optional
+
+    Returns
+    -------
+    list
+        List of unicode strings.
+
+    Examples
+    --------
+    >>> from gensim.parsing.preprocessing import preprocess_string
+    >>> s = "<i>Hel 9lo</i> <b>Wo9 rld</b>! Th3     weather_is really g00d today, isn't it?"
+    >>> preprocess_string(s)
+    [u'hel', u'rld', u'weather', u'todai', u'isn']
+
+    >>> from gensim.parsing.preprocessing import preprocess_string, strip_tags, strip_punctuation
+    >>> s = "<i>Hel 9lo</i> <b>Wo9 rld</b>! Th3     weather_is really g00d today, isn't it?"
+    >>> CUSTOM_FILTERS = [lambda x: x.lower(), strip_tags, strip_punctuation]
+    >>> preprocess_string(s, CUSTOM_FILTERS)
+    [u'hel', u'9lo', u'wo9', u'rld', u'th3', u'weather', u'is', u'really', u'g00d', u'today', u'isn', u't', u'it']
+
+    """
     s = utils.to_unicode(s)
     for f in filters:
         s = f(s)
     return s.split()
 
 
 def preprocess_documents(docs):
+    """Take list of strings, then apply default filters to every string.
```

Comment:
Redundant "Take" again. Also, refer to `docs` as documents here, not "list of strings". Write something like "Apply default filters to the documents strings."